parakeet/auth.py

```diff
@@ -78,6 +76,7 @@ def redirect_to_home(self):
     def click_sign_in(self):
         LOG.debug('click_sign_in')
         self.browser.find_element_by_xpath('//md-card-actions/button').click()
+        time.sleep(1)
```

Comment:
Do not use time.sleep as it slows down the tests.

Reply:
This is a workaround to avoid the Blank screen of OAuth. I figured that running any code that would interact with the new open window was causing a disturb on the proper loading of it. I don't know if there's a better way of doing that, just found this one to be the quicker. The blank page is a blocker for the tests btw.